handler/metadata.go

```diff
@@ -154,8 +152,8 @@ func (h *XlsxCreate) AddMetaDataToExcelStructure(ctx context.Context, excelInMem
 
 	processMetaElement("Version", strconv.Itoa(meta.Version.Version), true)
 
-	re := regexp.MustCompile("https://([^/]+)")
-	processMetaElement("Dataset Version URL", re.ReplaceAllString(meta.DatasetLinks.LatestVersion.URL, cfg.ExternalPrefixURL), true)
+	re := regexp.MustCompile("http(s)?://([^/]+)")
```

Comment:
Are the grouping parenthesis needed? Would the following work?
```suggestion
	re := regexp.MustCompile("https?://([^/]+)")
```

Reply:
Yep that works